code/game/objects/structures/window.dm

```diff
@@ -179,7 +179,7 @@ var/global/wcCommon = pick(list("#379963", "#0d8395", "#58b5c3", "#49e46e", "#8f
 	attack_generic(user, rand(10, 15))
 
 
-/obj/structure/window/attackby(obj/item/weapon/W as obj, mob/living/user as mob, params)
+/obj/structure/window/attackby(obj/item/W as obj, mob/living/user as mob, params)

```

Comment:
I'm going to be a stickler and say this need to be `obj/item/I`.

I know that entails a lot more changes, but this kinda goes against the standard already established.